app/views/response/show_calibration_results_for_student.html.erb

```diff
@@ -1,101 +0,0 @@
-<h1>Your calibration results <%= " for" + @assignment.course.name if !@assignment.course.nil?%></h1>
```

Comment:
Hi Yang, I think there must be some mistakes of the beginning version we cloned. I think for our project 1608, we only needs to touch question relative rb files. I don't know why the line is removed...